dlc-manager/tests/ln_dlc_channel_execution_tests.rs

```diff
@@ -351,6 +372,17 @@ impl EventHandler for LnDlcParty {
                     .unwrap();
                 self.blockchain.broadcast_transaction(&spending_tx);
             }
+            Event::ChannelClosed { channel_id, .. } => {
+                if let Err(error) = self
+                    .sub_channel_manager
+                    .notify_ln_channel_closed(channel_id)
```

Comment:
Yes, I think at some point it might be better to have the LDK `ChannelManager` built within the `SubChannelManager` so that we can make sure that everything is properly setup, but I think this PR is already big enough :)

Reply:
Yeah, that sounds like a pretty big change!